pwiz_tools/Skyline/SettingsUI/ViewLibraryPepMatching.cs

```diff
@@ -732,63 +737,76 @@ public SrmDocument AddPeptides(SrmDocument document, ILongWaitBroker broker, Ide
             int processedPercent = 0;
             int processedCount = 0;
             int totalMatches = listMatches.Count;
+            selectedPath = null;
 
-            var listPeptides = new List<PeptideDocNode>();
-            var hasSmallMolecules = false;
-            foreach (var match in listMatches)
+            // Some .blib files provide protein accessions (understood as Molecule List Names fro small molecules).
+            // If those are provided we will use them as node names.
+            // TODO(bspratt) for now we will do this only for small molecules since it might be a surprise to proteomics users. We should revisit that decision.

```

Comment:
Maybe CONSIDER. You shouldn't use TODO for something you don't feel confident you will address for the next release. We should talk about this as a policy decision and run a pass where people either address all TODOs or convert the ones they are not going to address into CONSIDER.

Reply:
Agreed, thanks.